doc/nrf/app_dev/device_guides/nrf54l/kmu_basics.rst

```diff
@@ -0,0 +1,63 @@
+.. _ug_nrf54l_developing_basics_kmu:
+
+Introducion to KMU key provisioning
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54L devices are equipped with a Key Management Unit (KMU) that facilitates secure and confidential storage of keys.
+This feature is crucial not only for private keys but also for public keys, as the :ref:`KMU can directly transfer a key to the CRACEN RAM<ug_nrf54l_crypto_kmu_cracen_peripherals>`.
+Even when keys must pass through addressable RAM, the KMU significantly reduces the risk of key exposure.
+Therefore, you should use KMU for managing secrets whenever possible.
+
+Key Types
+*********
+
+Different types of keys, such as revocable and locked keys, serve distinct purposes and have unique policies associated with their use and management.
+
+Revocable keys
+==============
+
+Keys can be set as invalid for further use. 
```

Comment:
```suggestion
Revocable keys can be set as invalid for further use which prevents from using or provisioning again the key slots.
```

Reply:
Suggestion: ``(...) which prevents key slots from being used or provisioned again``.

Reply:
subject of invalidation is the key